websites/0-9/3388/metadata.json

```
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "356827694572503051",
+		"name": "xela#0049"
+	},
+	"service": "3388",
+	"description": {
+		"en": "3388 is a streaming service for movies and TV."
+	},
+	"url": [
+		"3388.to",
+		"www.3388.to"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/FhGHKTn.png",
```

Logo needs to be 512x512
```suggestion
	"logo": "https://i.imgur.com/h8N7txd.png",
```